pom.xml

```diff
@@ -149,6 +153,11 @@
                 <artifactId>jackson-module-jaxb-annotations</artifactId>
                 <version>${version.jackson}</version>
             </dependency>
+            <dependency>
```

Comment:
What is it that needs woodstox?

Reply:
It's a compile dependency for hadoop Configuration class via hadoop-common.

Reply:
I took the standard approach to updating hadoop dependencies for this, but it's worth noting that H3x does also provide shaded jars for its client api and transitive dependencies, which would ostensibly eliminate the need for this. What do you guys think about moving toward usage of the shaded jars instead?

Reply:
I think it's great that Hadoop has moved to shaded jars. For us, I'm not sure we're a dependency of anything else, so there's probably no need. The only place I can think of where we might benefit is the guava library when used within code that is on the iterator stack. Then we're tied to Accumulo's version. If we shade and relocate guava, we could use a newer version on tservers, although really the reverse should be true -- Accumulo should be doing that.